showcase/app/templates/components/form/select.hbs

```diff
@@ -22,42 +22,77 @@
         </C.Options>
       </Hds::Form::Select::Base>
     </SF.Item>
+    <SF.Item @label="Multiple">
+      <Hds::Form::Select::Base multiple as |C|>
```

Comment:
```suggestion
      <Hds::Form::Select::Base aria-label="multiple select" multiple as |C|>
```
[Issue] - The lack of an `aria-label` here is causing the automated a11y checks to fail.